internal/pkg/exec/docker.go

```diff
@@ -159,3 +161,38 @@ func imageName(uri, tag string) string {
 	}
 	return fmt.Sprintf("%s:%s", uri, tag)
 }
+
+// GetHelperProviderFromDockerCfg will fetch credential store used by docker from config file
+func (c DockerCommand) GetHelperProviderFromDockerCfg() (string) {
```

Comment:
Quick question: does this function work for windows? Since we support windows maybe we don't want this function to break Windows users.